coderadar-vcs/src/main/java/io/reflectoring/coderadar/vcs/adapter/DeleteRepositoryAdapter.java

```diff
@@ -0,0 +1,34 @@
+package io.reflectoring.coderadar.vcs.adapter;
+
+import io.reflectoring.coderadar.vcs.port.driven.DeleteRepositoryPort;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import org.apache.tomcat.util.http.fileupload.FileUtils;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.lib.Repository;
+import org.eclipse.jgit.storage.file.FileRepositoryBuilder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DeleteRepositoryAdapter implements DeleteRepositoryPort {
```

Comment:
Rename to `DeleteLocalRepositoryAdapter`

Reply:
Done